diff --git a/src/NvFetcher/NixExpr.hs b/src/NvFetcher/NixExpr.hs
index 022256b..80081b1 100644
--- a/src/NvFetcher/NixExpr.hs
+++ b/src/NvFetcher/NixExpr.hs
@@ -168,6 +168,7 @@ fetcherToDrv (toNixExpr -> fetcherExpr) (quote -> drvName) =
       src = $fetcherExpr;
       nativeBuildInputs = [ unzip ];
       dontBuild = true;
+      dontCheckForBrokenSymlinks = true;
       installPhase = ''
         mkdir $$out
         cp -r * $$out
